pyccel/codegen/pipeline.py

```diff
@@ -196,12 +212,65 @@ def handle_error(stage):
 
         #------------------------------------------------------
         # TODO: collect dependencies and proceed recursively
-    #    if recursive:
-    #        for dep in parser.sons:
-    #            # Call same function on 'dep'
-    #            pass
+        # if recursive:
+        #     for dep in parser.sons:
+        #         # Call same function on 'dep'
+        #         pass
         #------------------------------------------------------
 
+        # Iterate over the internal_libs list and determine if the printer
+        # requires an internal lib to be included.
+        for lib in internal_libs:
+            if lib in codegen.get_printer_imports():
+                # get the include folder path and library files
+                if lib not in internal_libs_name:
+                    # get the library folder name
+                    lib_name = internal_libs[lib]
+                    # get path to pyccel/stdlib/lib_name
+                    import pyccel.stdlib as stdlib_folder
```

Comment:
Is there any reason the import is not done at the top of the file?

Reply:
the import is needed just to get the path of `stdlib`.